lib/webpacker/package_json.rb

```diff
@@ -0,0 +1,58 @@
+# Loads the package.json file from app root to read the webpacker configuration
+
+class Webpacker::PackageJson
```

Comment:
Agree, I felt too, but later realised that's how most configuration files looks. Anything you think that can make this look simpler?